ports/esp8266/machine_uart.c

```diff
@@ -329,3 +329,5 @@ const mp_obj_type_t pyb_uart_type = {
     .protocol = &uart_stream_p,
     .locals_dict = (mp_obj_dict_t *)&pyb_uart_locals_dict,
 };
+
+MP_REGISTER_ROOT_POINTER(byte * uart0_rxbuf);
```

Comment:
Can you remove the space after `*` for formatting consistency?

Reply:
The CI code format check fails without the space.

Reply:
e.g. https://github.com/micropython/micropython/runs/7156973079?check_suite_focus=true

Reply:
OK